data/json/items/gun/44.json

```
@@ -177,5 +177,100 @@
+    "type": "GUN",
+    "name": { "str": ".44 long barrel revolver" },
+    "description": "A large frame double-action revolver with a modern plastic handle.  Too large for the law enforcement, it is mostly used by sport enthusiasts shooters and hunters, for manufacturer deemed it to be the most accurate .44 revolver in their line.  This one has extended 8 inch barrel.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "colt_anaconda_8",
+        "name": { "str": "Colt Anaconda" },
+        "description": "A large frame double-action revolver with a modern plastic handle.  Too large for the law enforcement, it is mostly used by sport enthusiasts shooters and hunters, for manufacturer deemed it to be the most accurate .44 revolver in their line.  This one has extended 8 inch barrel."
+      }
+    ],
+    "weight": "1672 g",
+    "volume": "1384 ml",
+    "//": "350 x 153 x 70, but trimmed massively",
+    "longest_side": "350 mm",
+    "barrel_length": "203 mm",
+    "price": "1499 USD",
+    "price_postapoc": "15 USD",
+    "material": [ "steel", "plastic" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": [ "44" ],
+    "dispersion": 280,
+    "durability": 8,
+    "blackpowder_tolerance": 56,
+    "clip_size": 6,
+    "valid_mod_locations": [
+      [ "barrel", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 2 ],
+      [ "rail", 1 ],
+      [ "sights", 1 ],
+      [ "stock", 1 ],
+      [ "underbarrel", 1 ]
+    ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE",
+        "rigid": true,
+        "ammo_restriction": { "44": 6 },
+        "allowed_speedloaders": [ "44_speedloader6" ]
+      }
+    ],
+    "melee_damage": { "bash": 10 }
+  },
+  {
+    "id": "colt_anaconda_6",
+    "copy-from": "colt_anaconda_8",
+    "type": "GUN",
+    "name": { "str": ".44 revolver" },
+    "description": "A large frame double-action revolver with a modern plastic handle.  Too large for the law enforcement, it is mostly used by sport enthusiasts shooters and hunters, for manufacturer deemed it to be the most accurate .44 revolver in their line.  This one has 6 inch barrel.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "colt_anaconda",
+        "name": { "str": "Colt Anaconda" },
+        "description": "A large frame double-action revolver with a modern plastic handle.  Too large for the law enforcement, it is mostly used by sport enthusiasts shooters and hunters, for manufacturer deemed it to be the most accurate .44 revolver in their line.  This one has 6 inch barrel."
+      }
+    ],
+    "weight": "1499 g",
+    "volume": "1299 ml",
+    "//": "300 x 153 x 70, but trimmed massively",
+    "longest_side": "300 mm",
+    "barrel_length": "153 mm",
+    "price": "1499 USD",
+    "symbol": "(",
+    "color": "dark_gray"
+  },
+  {
+    "id": "colt_anaconda_4",
+    "copy-from": "colt_anaconda_8",
+    "type": "GUN",
+    "name": { "str": ".44 long barrel revolver" },
```

cough cough.

---

there is 2 inch version, so technically it is not entirely false
```suggestion
    "name": { "str": ".44 short barrel revolver" },
```